src/actionobserver.cpp

```diff
@@ -143,38 +143,43 @@ ActionObserver::ActionObserver(QObject *parent) :
     _actionObserver(new QMailActionObserver(this))
 {
     connect(_actionObserver, SIGNAL(actionsChanged(QList<QSharedPointer<QMailActionInfo> >)),
-        this, SLOT(actionsChanged(QList<QSharedPointer<QMailActionInfo> >)));
+            this, SLOT(actionsChanged(QList<QSharedPointer<QMailActionInfo> >)));
 }
 
 // Report only long sync type of actions.
 // Small actions like exporting updates and flags are ignored, same for actions
 // that only happen on email client UI is visible(dowload inline images, message parts, ...)
 bool ActionObserver::isNotificationAction(QMailServerRequestType requestType)
 {
-    if (requestType == TransmitMessagesRequestType || requestType == RetrieveFolderListRequestType
-            || requestType == RetrieveMessageListRequestType || requestType == RetrieveMessagesRequestType
-            || requestType == RetrieveMessageRangeRequestType || requestType == RetrieveAllRequestType
-            || requestType == SynchronizeRequestType || requestType == RetrieveNewMessagesRequestType) {
-        return true;
-    }
-    return false;
+    return requestType == TransmitMessagesRequestType
+            || requestType == RetrieveFolderListRequestType
+            || requestType == RetrieveMessageListRequestType
+            || requestType == RetrieveMessagesRequestType
+            || requestType == RetrieveMessageRangeRequestType
+            || requestType == RetrieveAllRequestType
+            || requestType == SynchronizeRequestType
+            || requestType == RetrieveNewMessagesRequestType;
 }
 
 // ################ Slots #####################
 
 void ActionObserver::actionsChanged(QList<QSharedPointer<QMailActionInfo> > actionsList)
 {
-    foreach(QSharedPointer<QMailActionInfo> action, actionsList) {
+    foreach (QSharedPointer<QMailActionInfo> action, actionsList) {
         // discard actions already in the queue and fast actions to avoid spamming transfer-ui
         if (!_runningActions.contains(action.data()->id()) && isNotificationAction(action.data()->requestType())
                 && !_completedActions.contains(action.data()->id())) {
             RunningAction* runningAction = new RunningAction(action, _accountsCache, this);
             _runningActions.insert(action.data()->id(), runningAction);
-            connect(runningAction, SIGNAL(actionComplete(quint64)), this, SLOT(actionCompleted(quint64)));
+            connect(runningAction, SIGNAL(actionComplete(quint64)),
+                    this, SLOT(actionCompleted(quint64)));
```

Comment:
We could change signal-slot connections to function references to get compilation time errors out of these.

Reply:
Yea, initially just did it where i touched the related signals/slots, but let's just switch fully to new connection format and be done with it.